letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -458,7 +471,10 @@ def _create_vhost(self, path):
                 is_ssl = True
 
         filename = get_file_path(path)
-        is_enabled = self.is_site_enabled(filename)
+        if self.conf("handle-sites"):
+            is_enabled = self.is_site_enabled(filename)
+        else:
+            is_enabled = True
```

Comment:
Thanks for explaining. 

Is the following scenario possible ?
`letsencrypt-auto` is ran under a debian-like distro, and the user specifies `--apache-handle-sites False` (or something like that). So a vhost may not be enabled, and `conf("handle-sites")` is False. (A user may just want to see the vhost config generated by letsencrypt, before enabling it).

Semantically, I think that the check for `conf("handle-sites")` (or other checks that signifies a non debian-like underlying distro) should be encapsulated in `self._is_site_enabled(...)`.